flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/factories/DataGenOptionsUtil.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.factories;
+
+import org.apache.flink.annotation.Internal;
+
+/** Utilities for {@link DataGenOptions}. */
+@Internal
+public class DataGenOptionsUtil {
```

Comment:
Actually, the task of the `Factory` is to translate options. We could also get rid of most of util classes and move the code to the factory itself.

Reply:
For DataGen I'm hesitant to do so because the factory is marked as PublicEvolving (and these would have to be visible because they're accessed from other places, too). Most other factories (of "real connectors") are marked Internal, though. I'm not sure if we can just changes this to Internal now? If not I'd probably keep it separate for now.